TOOLS/umpv

```diff
@@ -31,9 +31,6 @@ import os
 import socket
 import errno
 import subprocess
-import fcntl
-import stat
-import string
```

Comment:
Line 46: `    allowed_symbols = string.ascii_letters + string.digits + '_'`

Reply:
The weird thing is that it ran without exceptions!..i will fix that and squash..

Reply:
Who about now?